docs/streams/subscribe-streams.md

```diff
@@ -0,0 +1,270 @@
+---
+sidebar_position: 70
+title: Subscribe to Streams
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page explains how to publish messages to a stream in Macrometa.
```

Comment:
We are not "publishing" but subscribing here right? ..
Also yes I didn't find any subscribe to stream CLI command too..

Reply:
Thanks for catching that!